### PR TITLE
style(gh-pages): Changes to sidebar

### PR DESCRIPTION
**Pull request by fshovchko**
I made following changes in this PR:
-Removed All menu item from sidebar;
-Fixed focus on sidebar heading;
-Fixed sidebar script for IE.

### CODE REVIEW

**Review by dshovchko — APPROVED** [review]

File: pages/views/_includes/navigation/sidebar.njk [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
     sidebar.querySelector('.sidebar-trigger-btn').removeAttribute('active');
     Array.from(sidebar.querySelectorAll('.sidebar-nav-secondary'))
-      .forEach((menu) => menu.classList.remove('open'));
+      .forEach(function (menu) {
```
Comment (abarmina) [inline pages/views/_includes/navigation/sidebar.njk:38]: very cosmetic: extra space

```suggestion
      .forEach(function(menu) {
```

File: pages/src/navigation/sidebar/sidebar.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ export class ESLDemoSidebar extends ESLToggleable {
 
   public expandActive(noCollapse: boolean = false) {
     this.$submenus
-      .filter((menu) => !!menu.querySelector(this.activeLinkSel))
+      .filter((menu) => !!menu.hasAttribute('data-open'))
```
Comment (abarmina) [inline pages/src/navigation/sidebar/sidebar.ts]:
```suggestion
      .filter((menu) => menu.hasAttribute('data-open'))
```

**Review by abarmina — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Pull request merged**